File: 백준/Bronze/2525. 오븐 시계/오븐 시계.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
let input = fs.readFileSync(filePath).toString().split("\n");

let clock = input[0].split(" ");
let timer = input[1];

solution(parseInt(clock[0]), parseInt(clock[1]), parseInt(timer));

function solution(clockHour, clockMin, timer) {
  let plusClockMin = clockMin + timer;
  let restMin = 0;
  if (plusClockMin >= 60) {
    clockHour += parseInt(plusClockMin / 60);
  }

  if (plusClockMin >= 60) {
    restMin = plusClockMin % 60;
  } else {
    restMin = plusClockMin;
  }

  if (restMin == 60) {
    restMin = 0;
  }

  if (clockHour == 24) {
    clockHour = 0;
  } else if (clockHour > 24) {
    clockHour = parseInt(clockHour % 24);
  }
  console.log(`${clockHour} ${restMin}`);
}

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
H,M = map(int,input().split())
TM = int(input())

minute = M + TM
if minute >= 60 :
  hour = ((M + TM) // 60) + H
  minute = ((M + TM) % 60)
else :
  hour = H

if hour >= 24 :
  hour = hour -24
print(str(hour) + " " + str(minute))